{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/cronjob.yaml

```diff
@@ -18,11 +18,20 @@ spec:
             command: ['/bin/bash', '-c']
             args:
               - python manage.py dumpdata -o backups/db.json ;
-              - cat backups/db.json
+                echo backups/db.json ;
+                cat backups/db.json
```

Comment:
Maybe we should use an absolute path (also above in the python command) to make the location more explicit? (Really just a question...)

Reply:
yes, that would be clearer.  OK, I can make that change.